pro/pro-users.rst

```diff
@@ -1,194 +1,82 @@
 
-They will then be at their Dashboard, and will need to select an option. Since they are Level 2, they have only one option, "Remote Access:"
+Attaching or removing user from a client gateway
+================================================
+To attach users to a client gateway or remove users from a gateway, you will need to have the client gateway set up. 
+Once the client gateway is set up, you will see an option to remove users from the dropdown menu.
 
-.. image:: images/users/user-dashboard-2.png
+.. image:: images/users/gateway-dropdown.png
    :width: 80%
-   :alt: Remote Access 1
+   :alt: Add Remove User dropdown
    :align: center
 
-Clicking this brings them to the familiar "ext clients" screen. Here, they can create, manage, delete, view, and download their own ext clients for any Ingress Gateways on the network. At the top, it shows how many clients they have used, as opposed to their limit.
+You can click the button to either attach or remove a user
 
-.. image:: images/users/user-dashboard-3.png
+.. image:: images/users/attach-remove-users.png
    :width: 80%
-   :alt: Remote Access 2
+   :alt: Attach Remove User Modal
    :align: center
 
-After creating 2 clients, the user will be unable to create any more (in this example, their limit was set to "2").
-
 
-.. image:: images/users/user-dashboard-4.png
-   :width: 80%
-   :alt: Remote Access 3
-   :align: center
+Transferring super admin rights
+===============================
 
+Super admin rights can be transferred only to another admin. To do this, on the users page, go to the superadmin row and hover over the ellipsis.
+In the dropdown menu, an option to transfer admin rights should be present. On clicking it, a dialog box will open allowing you to select any admin 
+to transfer super admin rights to.
 
-Level 1: Node Access 4
-------------------------
-
-Now let's modify this user's Access Level to 1. Now, when they log in, they will have two options in their panel. Remote Access, and Nodes.
-
-.. image:: images/users/user-dashboard-5.png
+.. image:: images/users/transfer-super-admin-rights.png
    :width: 80%
-   :alt: Node User 1
+   :alt: transfer super admin rights
    :align: center
 
-This user can create and delete nodes in the network, up to their limit. To do this, they join via SSO (or username/password):
+Using the remote access client
+==============================
+Once a user has been added to a client gateway, they can connect to a network using the remote access client. To do this, they will first need to log in using the credentials that were provided to them.
 
-.. image:: images/users/user-dashboard-7.png
+.. image:: images/users/remote-access-client.png
    :width: 80%
-   :alt: Node User 2
+   :alt: Remote access login
    :align: center
 
-.. image:: images/users/user-dashboard-8.png
-   :width: 80%
-   :alt: Node User 3
-   :align: center
+Then they will have to select a network and subsequently a client gateway to use and after click on connect
```

Comment:
`Then they will have to select a network and subsequently a client gateway to use and after click on connect`. -> `After successful login you will be shown all the networks and gateways you have given access to, so now you will be able to connect/disconnect/refresh your connection to a gateway`